x-pack/solutions/security/packages/security-ai-prompts/README.md

```diff
@@ -1,3 +1,148 @@
+   ```
+
+3. **In a new terminal, the `which` command should be able to find it:**
+
+   ```bash
+   ➜  ~ which elastic-package
+   /Users/stephmilovic/dev/elastic-package/elastic-package
+   ```
+
+4. **Build `elastic-package` from the root before you begin:**
+
+   ```bash
+   ➜  ~ pwd
+   /Users/stephmilovic/dev/elastic-package
+   ➜  ~ make build
+   ```
+   
+---
+
+### Developer Flow: Updating Saved Objects Content
+
+1. **Generate the latest prompt files in Kibana:**
+
+   ```bash
+   cd $KIBANA_HOME/x-pack/solutions/security/plugins/elastic_assistant
+   yarn generate-security-ai-prompts
+   ```
+
+2. **Copy the generated prompt files into the integration package:**
+
+   ```bash
+   cd $INTEGRATIONS_HOME/packages/security_ai_prompts/kibana/security_ai_prompt
+   rm ./*.json
+   cp $KIBANA_HOME/target/security_ai_prompts/*.json .
+   ```
+
+---
+
+### Bump the Package Version
+
+1. **Open the `manifest.yml` for the integration:**
+
+   ```bash
+   cd $INTEGRATIONS_HOME/packages/security_ai_prompts
+   ```
+
+2. **Increment the `version:` field in `manifest.yml` following semantic versioning**  
+   _(e.g., `0.0.1` → `0.0.2`)_
+
+   > Use a **patch bump** for content-only changes like updated saved objects.
+
+---
+
+### Validate & Build the Integration Locally
+
+1. **Run linting and build the package:**
+
+   ```bash
+   ➜  ~ pwd 
+   /Users/stephmilovic/dev/integrations/packages/security_ai_prompts
+   
+   elastic-package lint
+   elastic-package build
+   ```
+
+2. **Restart your local Elastic Stack with the updated package:**
+
+   ```bash
+   elastic-package stack down
+   elastic-package stack up -d -v --services package-registry
+   ```
+
+---
+
+### Test the Updated Package
+
+1. Confirm the saved objects are included and working as expected by navigating to your Kibana instance and testing the integration. Ensure the following settings are in `kibana.dev.yml`:
+   ```
+   xpack.fleet.internal.registry.kibanaVersionCheckEnabled: false
+   xpack.fleet.registryUrl: https://localhost:8080
+   # Must match format_version in manifest.yml
+   xpack.fleet.internal.registry.spec.max: '3.4'
+   ```
+
+2. Start Kibana with:
+   ```
+   NODE_EXTRA_CA_CERTS=~/.elastic-package/profiles/default/certs/kibana/ca-cert.pem yarn start --no-base-path
+   ```
+
+3. Visit the Integration page and find Security AI Prompts (you may need to toggle Display beta integrations). Install it. Then go to the saved objects are to verify the saved objects have been installed and are what you expect. At this time, you can test your AI flow to ensure the updated prompt is being used, or use dev_tools to query for the prompt. For example:
+   ```
+   GET .kibana/_search
+   {
+     "query": {
+       "bool": {
+         "must": [
+           { "term": { "type": "security-ai-prompt" }},
+           { "term": { "security-ai-prompt.promptId": "defendInsights-incompatibleAntivirusRefine" }}
+         ]
+       }
+     }
+   }
+   ```
+
```

Comment:
Could we please add a section that instructs to run evals before and after to ensure that there is no regression in functionality?